src/initialization/initializeTypeScript/initializeTests.ts

```diff
@@ -15,6 +16,7 @@ const initializeBuiltInTests = async () => {
         "src/**/__tests__/*.test.{ts,tsx}",
         "src/**/*.test.{ts,tsx}",
         "test/**/*.{ts,tsx}",
+        "none",
```

Comment:
Nit: I kind of like this a bit more because it's a little more clear that it's not a real path. What do you think?

```suggestion
        "(none)",
```